webextensions/api/storage.json

```diff
@@ -246,6 +246,26 @@
                 }
               }
             }
+          },
+          "setAccessLevel": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/setAccessLevel",
+              "support": {
+                "chrome": {
+                  "version_added": "102"
```

Comment:
Worth adding a note that it's only supported for the session storage area, since the intent was to eventually offer it in more areas.